core/src/test/java/org/apache/iceberg/TestTables.java

```diff
@@ -93,6 +93,26 @@ public static TestTable create(
     return new TestTable(ops, name, reporter);
   }
 
+  public static TestTable create(
+      File temp,
+      String name,
+      Schema schema,
+      PartitionSpec spec,
+      int formatVersion,
+      Map<String, String> properties) {
```

Comment:
There was no option to pass the table properties before. 
Needed to pass different file format for paramterized test.